uaclient/tests/test_cli_collect_logs.py

```diff
@@ -0,0 +1,107 @@
+import textwrap
+
+import mock
+import pytest
+
+from uaclient.cli import (
+    action_collect_logs,
+    collect_logs_parser,
+    get_parser,
+    main,
+)
+from uaclient.exceptions import NonRootUserError
+
+M_PATH = "uaclient.cli."
```

Comment:
I'm not sure if the `M_PATH` use is necessary for any of the mocks here

Reply:
Well I think it is always good to mock stuff where it is imported, right?